### ISSUE TITLE
Indexing tz..-accounts (wallets)

### ISSUE THREAD

**Issue by droserasprout** [dipdup-io/dipdup#291]
**What feature would you like to see in DipDup?**

Indexing tz..-accounts (wallets)

**Why do you need this feature, what's the use case?**

Requested by Veqtor and nicoarbogast#0001 in Discord
Also needed for executors feature.

**Is there a workaround currently?**

No.

**Comment by droserasprout** [comment]
DipDup's main focus is still smart contracts. And we gave up on PyTezos executors. Closing?

**Comment by m-kus** [comment]
@veqtor is it still actual for you?

**Comment by veqtor** [comment]
> @veqtor is it still actual for you?

Yes very much, maybe a special type of index could work that creates a handler for transfers?

**Comment by m-kus** [comment]
Implement together with #572
